src/api/table.ts

```diff
@@ -18,11 +18,14 @@ export const updateVenueTable = async ({
   subtitle,
   capacity,
 }: UpdateVenueTableProps) => {
+  const updatedCapacity = parseInt(capacity.toString(), 10);
+  const tableColumns = tableOfUser?.columns ?? 1;
```

Comment:
Why is the default `1`? This should be imported from the `settings.ts`, if it doesn't exist there, then we should add it.

Reply:
replaced by MIN_TABLE_CAPACITY, fixed in 2bd967a